### DISCUSSION TITLE
Sending data to a server

### DISCUSSION THREAD

**Discussion by emullallphotography** [community/community#69419] [Programming Help]
### Body

Hi,

I am coding a website for HWDSB that emails students about overdue work and I was wondering if it would work if I send a request to the HWDSB website to get overdue work than output it as HTML on my website using PHP and MySQL and finally send the student an email about it.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by AlexanderMelde** [comment]
Is HWDSB the name of the school? 
It should be possible either using a webscraper or an official API of the schools website.

The basic idea would be:

Every x minutes:
1) Download the page with all the information
2) Write a script to parse the information, filter out everything irrelevant
3) If relevant Info changed since the last parsing:
      3.1) Optional: Write the relevant information into a own database
      3.2) Send relevant info via email

If you decide to use a scraper, please keep in mind that frequent use may be considered as DDOS attack, which can be illegal and should be accepted by the school first (contact them for permission).

**Reply by ghost** [reply]
HWDSB is the school board

**Reply by ghost** [reply]
How would I calculate every X minute so it sends it fast enough but not too fast?
